### DISCUSSION TITLE
Attenzione utenti Home Assistant OS: versione Core 2021.9 dà potenziali problemi

### DISCUSSION THREAD

**Discussion by indomus** [indomus/forum#673] [HUB: Home Assistant]
ATTENZIONE, per tutti gli utilizzatori di Home Assistant OS su Raspberry Pi 3 e ODROID-C2: si consiglia di NON aggiornare alla versione 2021.9, dato che in alcuni casi tale aggiornamento pare causare un continuo reboot dell’host.
Commentate liberamente qui, in italiano, se avete problemi.

Il thread ufficiale: https://github.com/home-assistant/core/issues/55541

**Comment by riccionese** [comment]
Non so se c'entra ma anche io con Pi4 e core ho dovuto fare downgrade che HA non ripartiva

**Reply by sky7176** [reply]
Buon giorno,
il mio sistema non mi segnala ancora la disponibilità dell'aggiornamento

Giuseppe

**Reply by snakuzzo** [reply]
Perché c'è un problema su HassOS...e nello specifico su RPI3x
Leggi [QUI](https://github.com/home-assistant/core/issues/55541)

**Reply by sky7176** [reply]
Grazie

**Reply by genus-giuseppe** [reply]
anche io non vedo l'aggiornamento ho istallazione su proxmox

**Comment by indomus** [comment]
Problema rientrato con versione 2021.9.1

https://github.com/home-assistant/core/issues/55541#issuecomment-912022141
